2024-04-03 16:58:09,180 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-04-03 16:58:09,180 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /Users/elvis/Documents/External-Work/Web3Suite/web3suite/suite-env/lib/python3.9/site-packages/instabot/bot/bot.py
2024-04-03 16:58:09,180 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-04-03 16:58:09,440 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2024-04-03 16:58:09,440 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2024-04-03 16:58:09,440 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2024-04-03 16:58:09,440 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
